demo/public/index.html

```diff
@@ -1,11 +1,27 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <!-- Include gtag... data layer commands conflict with measure though -->
     <script async src="https://www.googletagmanager.com/gtag/js?id=%REACT_APP_GA_TRACKING_ID%"></script>
     <script>
-      window.dataLayer = window.dataLayer || [];
-      function gtag(){dataLayer.push(arguments);}
-      gtag('js', new Date());
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+    </script>
+    <!-- Include measurement library. Normally we can't run this and gtag on
+         the same site, because both use the same dataLayer object and send
+         conflicting events. As a workaround, we do not run async and include
+         a measure-setup compiled file that does not run commands on load,
+         then call it in the setup script. -->
+    <script src="./measure-no-config.js"></script>
+    <script>
+      measureDataLayer = [];
+      window.setupMeasure(measureDataLayer);
+      function measure(){measureDataLayer.push(arguments);}
+      measure("config", "googleAnalytics", {
+            "measurement_id": "%REACT_APP_GA_TRACKING_ID%",
```

Comment:
I actually would prefer using a separate ID, so that we can more easily compare which events are from ML and which are from standard gtag.  I'll set up a property for this.

Reply:
Oh, this is a great idea actually. I don't see any property right now but once it exists I will update the PR